shared/src/business/useCases/trialSessions/saveCalendarNoteInteractor.test.js

```diff
@@ -0,0 +1,185 @@
+import { applicationContext } from '../../test/createTestApplicationContext';
+import { saveCalendarNoteInteractor } from './saveCalendarNoteInteractor';
+const {
+  MOCK_CASE,
+  MOCK_CASE_WITH_TRIAL_SESSION,
+} = require('../../../test/mockCase');
+const {
+  ROLES,
+  TRIAL_SESSION_PROCEEDING_TYPES,
+} = require('../../entities/EntityConstants');
+
+describe('saveCalendarNotes', () => {
+  let mockCurrentUser;
+  let mockTrialSession;
+  let mockCase;
+
+  const MOCK_TRIAL = {
+    caseOrder: [],
+    maxCases: 100,
+    proceedingType: TRIAL_SESSION_PROCEEDING_TYPES.remote,
+    sessionType: 'Regular',
+    startDate: '2025-12-01T00:00:00.000Z',
+    term: 'Fall',
+    termYear: '2025',
+    trialLocation: 'Birmingham, Alabama',
+    trialSessionId: '8675309b-18d0-43ec-bafb-654e83405412',
+  };
+
+  beforeEach(() => {
+    mockCurrentUser = {
+      role: ROLES.docketClerk,
+      userId: '8675309b-18d0-43ec-bafb-654e83405411',
+    };
+
+    mockTrialSession = MOCK_TRIAL;
+
+    mockCase = MOCK_CASE;
```

Comment:
Couple of other instances where if we change `mockTrialSession` or `mockCase` in a test, it will mutate the original test data.

Reply:
Good catch!

Reply:
Updated!